content/intro-to-storybook/angular/en/simple-component.md

```diff
@@ -182,18 +157,31 @@ Once we’ve done this, restarting the Storybook server should yield test cases
   />
 </video>
 
+## Specify data requirements
+
+It’s best practice to specify the shape of data that a component expects. Not only is it self documenting, it also helps catch problems early. Here, we'll use Typescript and create a interface for the `Task` model.
+
+Create a new folder called `models` inside `app` folder and inside a new file called `task.model.ts` with the following content:
+
+```typescript
+export interface Task {
+  id: string;
+  title: string;
+  state: string;
+}
+```
+
 ## Build out the states
 
 Now we have Storybook setup, styles imported, and test cases built out, we can quickly start the work of implementing the HTML of the component to match the design.
 
-The component is still basic at the moment. First write the code that achieves the design without going into too much detail:
+Our component is still rather rudimentary at the moment. We're going to make some changes so that it matches the intended design without going into too much detail:
 
 ```typescript
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
```

Comment:
I think this line should be kept. Without it my IDE throws errors on all the instances where we use these decorators, which gives the appearance that the code is broken (I think the code would still work, but in Angular the best practice is to include these imports.

Reply:
@alexanderdodd this is fixed with the latest commit. Also i updated the repo i used for the documentation. As with Storybook version 5.3 some significant changes come into play and with that it would be best to start from a "clean slate" (pardon the bad pun), if you click the link in the description it should point to the updated repository